src/cmd/config.go

```diff
@@ -70,21 +73,34 @@ func init() {
 	viper.BindPFlags(configSampleCmd.Flags())
 }
 
-func LoadConfig() (*common.Config, error) {
-	v := common.ConfigVersion{}
-	if err := viper.Unmarshal(&v); err != nil {
-		return nil, err
+func readConfig() []byte {
```

Comment:
**IMPORTANT NOTE**

For some reason the old code tried to read a config file from `$HOME/opslevel-k8s.yaml`. I think that might have been a mistake or some old legacy code we didn't update. Currently `cfgFile` is set from the command line and its default is `./opslevel-k8s.yaml`. 

New logic makes no attempt to read from `$HOME/opslevel-k8s.yaml`, unless the user specifically asked for it using `-config`.